frontend/src/components/panels/Document.vue

```diff
@@ -232,6 +232,7 @@ import OptionalInput from "@/components/widgets/inputs/OptionalInput.vue";
 import ToolOptions from "@/components/widgets/options/ToolOptions.vue";
 import { SectionsOfMenuListEntries } from "@/components/widgets/floating-menus/MenuList.vue";
 
+const FILE_SUFFIX = ".graphite";
```

Comment:
Could you please move this (and `.svg`) into `consts.rs` and do the concatenation in the editor backend, for both `.graphite` and `.svg`? (So the concatenated name would be passed along in the `name` variable of the frontend messages.)